modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/search/CommerceSuggestionsContributor.java

```diff
@@ -0,0 +1,306 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.commerce.product.internal.search;
```

Comment:
I think this class should be moved to a package _spi.suggestions_
```suggestion
package com.liferay.commerce.product.internal.search.spi.suggestions;
```

see the reference from the search module:
_com.liferay.portal.search.spi.suggestions.SuggestionsContributor;_

Reply:
I've changed it removing the model from the package name so it is now com.liferay.commerce.product.internal.search.spi.suggestions, see: https://github.com/liferay-commerce/liferay-portal/pull/4784/commits/d3ca17e60f402e44ff24b4016a609784bceb59ed#diff-9da0c8c9a652d397812f3f5becbde61a6630472207a571fa7abc8c750f62a3bd

Unfortunately I don't think moving the JS contributor will be possible since it'll cause issues with the default search bar finding it.